### ISSUE TITLE
convert SRT to PAC character missing after newline on Ubuntu via CLI 

### ISSUE THREAD

**Issue by PunchaiW** [SubtitleEdit/subtitleedit#7321]
Hello,

I have a problem when convert SRT to PAC file

My goal : SRT -> PAC -> SRT

Here my scenario : I convert SRT to PAC with /pac-codepage:Latin and character after newline got missing

This is input SRT
![image](https://github.com/SubtitleEdit/subtitleedit/assets/77190490/3e343d94-a7a5-4299-96a3-4cec6a1a72c5)

This is SRT -> PAC convert on [ Windows ] opened by sublime with encoding Western (Windows 1252)
![image](https://github.com/SubtitleEdit/subtitleedit/assets/77190490/4eaee393-9cf8-4397-b459-cc08206da03a)

This is SRT -> PAC convert on [ Ubuntu ] opened by sublime with encoding Western (Windows 1252)
![image](https://github.com/SubtitleEdit/subtitleedit/assets/77190490/92127161-5253-46a9-b8f3-bb0bf50c5a5d)
xvfb-run -a mono SubtitleEdit.exe /convert /yourinputpath/subtitle_file.srt PAC /outputfilename:/youroutputpath/subtitle_file.pac /pac-codepage:Latin

Tested on SubtitleEditBeta(version 3.6.13) via CLI

How do i solve this?

Thank you!

**Comment by niksedk** [comment]
You could also try: https://github.com/SubtitleEdit/subtitleedit-cli

**Comment by PunchaiW** [comment]
I'm not sure if this is the correct way to solve 

here I insert temp character for missing instead the actual one  
![image](https://github.com/SubtitleEdit/subtitleedit/assets/77190490/a389cb22-9d09-4afc-9089-431092492d79)
Pac.cs Line 1925

This is just Latin, Test on Ubuntu

**Comment by PunchaiW** [comment]
Thank you for reply 

but it's seem newline got missing instead

This is input SRT
![image](https://github.com/SubtitleEdit/subtitleedit/assets/77190490/f1866771-dd28-44f4-be67-6136a8d2ea95)

This is PAC open by opened by sublime with encoding Western (Windows 1252)
![image](https://github.com/SubtitleEdit/subtitleedit/assets/77190490/8f66bbf0-ec04-4942-b35f-ad1d1b68186d)

This is SRT -> PAC -> SRT
![image](https://github.com/SubtitleEdit/subtitleedit/assets/77190490/08ccab6d-ade5-405a-90c7-d5243d1ef894)

i'm not sure how byte work but when it's cross OS 
Windows read newline as /r/n
Ubuntu read newline as /n
Does it cause byte difference?

**Comment by niksedk** [comment]
Yes, the issue is the the NewLine length difference... hopefully fixed now: https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.0/SubtitleEditBeta.zip
